addons/mail/static/src/core/web/base_recipients_list.js

```diff
@@ -19,16 +22,21 @@ export class BaseRecipientsList extends Component {
 
     /** @returns {Markup} */
     getRecipientListToHTML() {
-        const recipients = this.props.thread.recipients.slice(0, 5).map((
-            { partner }) => {
-                return `<span class="text-muted" title="${escape(
-                    partner.email || _t("no email address")
-                )}">${escape(partner.name)}</span>`;
-            });
+        const recipients = this.props.thread.recipients
+            .slice(0, 5)
+            .map(
```

Comment:
:socks: 
Please use available space to make code readable ...
```suggestion
        const recipients = this.props.thread.recipients.slice(0, 5).map(
```

Reply:
Actually is there any change ? Don't lint files in stable (especially for a bad linting)